spynnaker8/spynnaker_plotting.py

```diff
@@ -182,23 +195,23 @@ def plot_segment(axes, segment, label='', **options):
             heat_plot_neo(axes, segment.filter(name=name)[0], label=label,
                           **options)
     elif len(segment.spiketrains) > 0:
-        if len(segment.analogsignalarrays) > 1:
+        if len(analogsignals) > 1:
             raise Exception("Block.segment[0] has spikes and "
                             "other data please specifiy one "
                             "to plot")
         plot_spiketrains(axes, segment.spiketrains, label=label, **options)
-    elif len(segment.analogsignalarrays) == 1:
-        heat_plot_neo(axes, segment.analogsignalarrays[0], label=label,
+    elif len(analogsignals) == 1:
+        heat_plot_neo(axes, analogsignals[0], label=label,
                       **options)
-    elif len(segment.analogsignalarrays) > 1:
+    elif len(analogsignals) > 1:
         raise Exception("Block.segment[0] has {} types of data "
                         "please specify one to plot using name="
-                        "" % len(segment.analogsignalarrays))
+                        "" % len(analogsignals))
     else:
         raise Exception("Block does not appear to hold any data")
 
 
-class SpynakkerPanel(object):
+class Spynnakerpanel(object):
```

Comment:
Why did the name of this class change? It causes quite a few “noise” changes elsewhere

Reply:
Look closely as spelling. While I hide in shame.